### DISCUSSION TITLE
pikepdf >=3 needed for overlay/underlay/merge support

### DISCUSSION THREAD

**Discussion by lsces** [pdfarranger/pdfarranger#1128] [Q&A]
I have been hitting pdf Arranger pretty hard splitting  A3 scans of Model Engineer magazines back into page order. It does the job effortlessly and being able to see the whole mag at once allows me to check if any pages need rescanning. The problem I hit though is trying to merge two A4 pages back to one, where I get the above message. pip install gives me a pikepdf of 9.3.0 so well above '3' ... so what am I missing?

**Comment by dreua** [comment]
Please create an issue with the "bug report" issue template so we get all important information at once. The about dialog shows the pikepdf version in use.
